### ISSUE TITLE
Use fakeroot instead of requiring root

### ISSUE THREAD

**Issue by mkesper** [tomuta/debian-mini-odroid-c1#9]
Hi,

I think it should it be possible to use https://wiki.debian.org/FakeRoot instead of requiring root for building.
Building kernels as root is strongly discouraged.

**Comment by tomuta** [comment]
I agree it is discouraged and I generally don't like the idea of doing stuff as root, either.  But I don't think you can mount block devices using fakeroot.  Root privileges aren't really needed to build the kernel, all they are needed for are for mounting the image file, formatting it, and copying over the data.

**Comment by mkesper** [comment]
How about separating make and "make install" (as root) then?

**Comment by tomuta** [comment]
That would be an option.
